lib/Dialect/TritonGPU/Transforms/AccelerateMatmul.cpp

```diff
@@ -781,7 +781,7 @@ static void decomposeMixedModeDotOp(ModuleOp mod, int computeCapability) {
       // promote operands for sm < 89 since fp8 mma is not natively supported
       // promote operands for sm >= 90 when mma is not v3
       if (!isNativeFP8 ||
-          (isNativeFP8 && (computeCapability == 89 || mmaLayout.isHopper())))
+          (isNativeFP8 && (computeCapability >= 89 || mmaLayout.isHopper())))
```

Comment:
I don't think the mma.sync with fp8 are supported on sm_90 or sm_100

Reply:
I believe these are supported in PTX (and are tested by test_simple_matmul with the `BLOCK_M, BLOCK_N, BLOCK_K = 32, 32, 32` configuration). Prior to adding the changes to MMAv2.cpp, these tests were failing on hopper

https://github.com/triton-lang/triton/blob/19277de9a45dce943b246cd0875221a3a644c06a/python/test/unit/language/test_matmul.py#L90-L97

Additionally, the PTX docs for `mma` specify:

```
.e4m3 and .e5m2 alternate floating point type mma operation requires sm_89 or higher.
```

Reply:
ah interesting, thanks for the insight! let me take a closer look and limit this to a smaller set of GPUs